packages/app/src/systems/SignUp/components/MnemonicWrite/MnemonicWrite.tsx

```diff
@@ -32,7 +32,7 @@ export function MnemonicWrite({
         width={129}
         height={116}
       />
-      <Header title="Enter your Recovery Phrase" />
```

Comment:
it's better to keep "your Recovery Phrase" ? in the prev screen also uses Recovery Phrase